flyway-core/src/main/java/org/flywaydb/core/internal/dbsupport/DbSupportFactory.java

```diff
@@ -131,7 +131,7 @@ public static DbSupport createDbSupport(Connection connection, boolean printInfo
         }
 
 		//Sybase ASE support
-        if (databaseProductName.startsWith("ASE")) {
+        if (databaseProductName.startsWith("Adaptive Server Enterprise")) {
```

Comment:
Is there actually a SAP ASE database that returns "ASE" or is it the same as Sybase ASE that returns "Adaptive Server Enterprise"?